src/components/Signup.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import { useNavigate, Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import useActions from '../hooks/useActions';
+
+const Signup = () => {
+  const [username, setUsername] = useState('Samrood');
+  const [email, setEmail] = useState('samrood@gmailexample22.com');
```

Comment:
Empty strings are better initial states for your username, email and password

Reply:
Thanks. Good catch. I have removed these default state.

Reply:
Awesome!